crates/ironrdp-web/src/session.rs

```diff
@@ -818,13 +813,13 @@ impl Session {
     }
 
     #[allow(clippy::unused_self)]
```

Comment:
nitpick:
```suggestion
    #[expect(clippy::unused_self)]
```

Likely this suppression is not needed anymore.

Reply:
Removed